Src/Support/Google.Apis.Core/Http/ConfigurableMessageHandler.cs

```diff
@@ -558,6 +574,18 @@ bool DisposeAndReturnFalse(IDisposable disposable)
                             }
                         }
 
+                        if (!errorHandled)
+                        {
+                            errorHandled = await CredentialHandleResponseAsync(new HandleUnsuccessfulResponseArgs
```

Comment:
Nit: I'd probably introduce a local variable for the HandleUnsuccessfulResponseArgs:

```csharp
var handlerArgs = new HandleUnsuccessfulResponseArgs
{
    Request = request,
    Response = response,
    TotalTries = NumTries,
    CurrentFailedTry = NumTries - triesRemaining,
    CancellationToken = cancellationToken
};
errorHandled = await CredentialHandleResponseAsync(handlerArgs).ConfigureAwait(false);
```